fmriprep/cli/parser.py

```diff
@@ -58,8 +58,11 @@ def _filter_pybids_none_any(dct):
     def _bids_filter(value):
         from json import loads
 
-        if value and Path(value).exists():
-            return loads(Path(value).read_text(), object_hook=_filter_pybids_none_any)
+        if value:
+            if Path(value).exists():
+                return loads(Path(value).read_text(), object_hook=_filter_pybids_none_any)
+            else:
+                raise parser.error(f"Path does not exist: <{value}>.")
```

Comment:
maybe a `FileNotFoundError` would be more helpful here?
```suggestion
            raise FileNotFoundError(f"Path does not exist: <{value}>.")
```

Reply:
I followed what is done elsewhere in the parser: 
https://github.com/nipreps/fmriprep/blob/962f0b2e26d522d87ed786f12766001aa65e9dfd/fmriprep/cli/parser.py#L20-L24
but I have no problem changing it.

Reply:
sure that works too, though if you are following that example the function should be converted into a partial:

https://github.com/nipreps/fmriprep/blob/c707938e1462e77b1c3a1216a855ceb0aba70845/fmriprep/cli/parser.py#L76-L78

Reply:
oh right, my mistake. Would there be any advantage of making it a partial for that specific option?

Reply:
i think its necessary to pass in the defined parser

Reply:
Have we resolved this discussion? Would it be worth adding a test to make sure that it works, or are there tests in the CI that are covering the necessary cases?

Reply:
I think the partial call got lost in the latest rebase